addons/website/static/src/js/content/website_root.js

```diff
@@ -254,6 +254,23 @@ var WebsiteRoot = publicRootData.PublicRoot.extend(KeyboardNavigationMixin, {
         var res = this._unslugHtmlDataObject('seo-object');
         ev.data.callback(res);
     },
+    /**
+     * Gives the given button a loading effect.
+     *
+     * @private
+     * @param {OdooEvent} ev
+     */
+    _onBtnSpinRequest: function (ev) {
```

Comment:
Extracted in 14.0 to https://github.com/odoo/odoo/pull/63417
Wait & see if we can remove it from this PR